### ISSUE TITLE
Assertion Failure on Mac Catalyst

### ISSUE THREAD

**Issue by jaylyerly** [ProxymanApp/atlantis#42]
I have incorporated Atlantis into my iOS app and it works fine running on the iPad simulator.  My app supports macOS via Catalyst, but when I build and run my Mac target, I hit an `assertionFailure` in the `injectURLSessionResume` method in NetworkInjector.swift.

Does Atlantis not support the Catalyst environment?

**Comment by NghiaTranUIT** [comment]
Hi, at the moment, we haven't support Catalyst app yet, but it's a good time that we do it.

Our team is working on it now 👍

**Comment by NghiaTranUIT** [comment]
Hey @jaylyerly 

Let check the master branch, it's fixed with #43 

```bash
pod 'atlantis-proxyman',  :git => 'https://github.com/ProxymanApp/atlantis.git', :branch => 'main'
```

<img width="1469" alt="Screen Shot 2020-11-26 at 13 46 49" src="https://user-images.githubusercontent.com/5878421/100317354-cb08b700-2fee-11eb-99ac-43d03dc7c761.png">

I will release the official build soon 👍

**Comment by NghiaTranUIT** [comment]
We've released Atlantis 1.4.2, which fully supports Mac Catalyst. Please update your Pod file 👍
